fe/fe-core/src/main/java/org/apache/doris/fs/FileSystemFactory.java

```diff
@@ -33,7 +38,7 @@ public static FileSystem get(StorageBackend.StorageType type, Map<String, String
         return get(type.name(), type, properties);
     }
 
-    public static FileSystem get(String name, StorageBackend.StorageType type, Map<String, String> properties) {
+    public static RemoteFileSystem get(String name, StorageBackend.StorageType type, Map<String, String> properties) {
```

Comment:
`StorageBackend.StorageType` can be removed

Reply:
Can not remove it for now, it denpendent on Stmt